.gitignore

```diff
@@ -1,2 +1,4 @@
 # This is by benjaoming's convention the directory to use when dev'ing locally
 build/
+build_src/
+dist/
```

Comment:
```suggestion
# Docker working directory (shared with docker containers)
dist/
# Docker CID files
*.cid
```

Reply:
Not entirely true, except for the `debian-test` case. I can add comments, if you think they're important.

Reply:
Ah okay, then nevermind the suggestion - I find it nice to annotate `.gitignore`, it's a nice way to quickly figure out why there is some weird directory taking up space that isn't tracked in Git.